.github/workflows/ci.yml

```diff
@@ -40,19 +40,19 @@ jobs:
         experimental:
         - false
         node-version:
-        - 16.x
```

Comment:
Don't remove the top version from here. Yesterday we discussed that it needs to be both 12+16 for all workers, remember?